Dockerfile

```diff
@@ -14,14 +14,9 @@ RUN apt-get update && apt-get install -y python-dev python-pip curl && apt-get b
 # Directory to copy in for build.  After building, cp this out -- to
 # the same place.  Otherwise pip will have to rebuild everything,
 # every time.
-
-RUN mkdir /function/ /cache/
-ADD dist /function
-ADD .binaris/.pycache /cache
-COPY docker/* /function/
-WORKDIR /function/
-
-# Re-use PIP cache between runs.  Do *NOT* copy this out!
-ENV XDG_CACHE_HOME=/cache/
-
-CMD [ "./install.sh" ]
+RUN pip install virtualenv
+COPY oldrequirements.txt /tmp/oldrequirements.txt
```

Comment:
Please keep in the same directory (e.g. `requirements.txt`, `requirements.txt.bk`, `requirements.txt.bk2`).  I don't care about the names, I don't particularly care how many you keep of each file; I _do_ care they're all in the same directory.

Also the use of `/tmp` as a working directory is problematic.  I'd prefer some (any) other directory.